src/main/java/com/spikes2212/command/genericsubsystem/commands/AccelerateGenericSubsystem.java

```diff
@@ -0,0 +1,87 @@
+package com.spikes2212.command.genericsubsystem.commands;
+
+import java.util.function.Supplier;
+
+import com.spikes2212.command.genericsubsystem.GenericSubsystem;
+
+import edu.wpi.first.wpilibj.Timer;
+
+/**
+ * This command accelerate the speed of a {@link GenericSubsystem} linearly so it will
+ * reach a wanted speed after a given time.
+ *
+ * @author Yuval Levy
+ */
+public class AccelerateGenericSubsystem extends MoveGenericSubsystem {
```

Comment:
please make this command to just accelerate the subsystem, and finish when it is at the needed speed. 
We can use command groups, and specifically the `andThen` method in command in order to make the subsystem stay at the desired speed afterwards.